skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Setup.kt

```diff
@@ -24,4 +24,14 @@ object Setup {
             // Not all platforms allow this.
         }
     }
-}
\ No newline at end of file
+}
+
+fun tryEnableUIScaling() {
```

Comment:
Lets move all global properties initialization in one place (`Setup.init`):

1. Move this to https://github.com/JetBrains/skiko/blob/master/skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Setup.kt. 

2. Change signature of `Setup.init`:
```
object Setup {
    fun init(
        noEraseBackground: Boolean = System.getProperty("skiko.rendering.noerasebackground") != "false",
        globalLAF: Boolean = System.getProperty("skiko.rendering.laf.global") == "true",
        useScreenMenuBar: Boolean = System.getProperty("skiko.rendering.useScreenMenuBar") != "false",
        autoLinuxDpi = System.getProperty("skiko.linux.autodpi") == "true"
    ) {
```

We can call it [here](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Application.desktop.kt;l=160?q=Application%20desktop), and in `AppWindow` (`companion { init {} }`), before any Swing initialization.

Probably we can try to call it in `companion { init {} }` block of `ComposeWindow` / `ComposeDialog`. But don't know if it will work.

`skiko.linux.autodpi` will be set automatically in our Gradle plugin. in IDEA plugin application or other Swing application we shouldn't override it (when our Gradle plugin isn't used)